run-tests.php

```diff
@@ -63,6 +63,7 @@ function main()
     if (getenv("TEST_PHP_WORKER")) {
         $workerID = intval(getenv("TEST_PHP_WORKER"));
         run_worker();
+
```

Comment:
This newline shouldn't be here.

Reply:
I added `'blank_line_before_statement' => false,` to the cs fixer config and ran it again.